CondCore/L1TPlugins/src/plugin.cc

```diff
@@ -207,3 +207,16 @@ REGISTER_PLUGIN(L1CaloGeometryRecord, L1CaloGeometry);
 using namespace l1t;
```

Comment:
I have to wait for Khristian on that one...  I seem to recall that would break something in our O2O.  Note that the new Cond Format classes do not (lesson learned) use the l1t:: namespace.  Let's clean this up during the debug cycle when Khristian is back.